src/views/enter-registered-office-address.njk

```diff
@@ -174,7 +174,13 @@
               <p>" + i18n.enterRegisteredOfficeAddressPage.publicInformationLine1 + "</p>"
           }) }}
 
-          {% include "includes/save-and-continue-button.njk" %}
+          {{ govukButton({
+            text: i18n.buttons.saveAndContinue,
+            attributes: {
+              "id": "submit",
+              "data-event-id": "manual-address-save-and-continue"
```

Comment:
i think it would be better to have the data-event-id actually in the "includes/save-and-continue-button.njk" file and have the value as "save-and-continue"
Otherwise, as we will need an event fired on each save and continue, we'll never end up using that include if we don't add the id to it.

Reply:
Using the includes button.